pkg/cmd/root/help.go

```diff
@@ -66,7 +66,7 @@ func rootHelpFunc(cmd *cobra.Command, _ []string) {
 		helpEntries = append(helpEntries, helpEntry{"", longText})
 	}
 	if isRootCmd(cmd) && viper.GetBool(constants.ConfigShowOctopus) {
-		helpEntries = append(helpEntries, helpEntry{"", output.Bluef("%s", constants.OctopusLogo)})
+		helpEntries = append(helpEntries, helpEntry{"", output.Cyanf("%s", constants.OctopusLogo)})
```

Comment:
This was a deliberate change to make the colours of the Octopus logo printed match our branding better